---
layout: default
title: Research
modified: 2016-02-12
permalink: /research/
---
### Our laboratory seeks to understand how bacterial populations evolve and adapt to colonize hosts and cause disease.
By studying evolution-in-action, in experimental populations and in ongoing infections, using the latest methods in genomic sequencing, we are identifying mechanisms of bacterial adaptation *in vitro* and *in vivo*. 

{% include toc.html %}

## Biofilms
We are focused on how bacteria form complex communities within biofilms and how individual cells perceive cues to attach or disperse.

We have learned that adaptation involves a sequence of mutations in predictable targets that leads to diversification into ecologically differentiated subpopulations. Often, these subpopulations interact synergistically, with certain types affecting host response and others enhancing resistance. In addition, we now routinely sequence the genomes of hundreds of bacterial isolates from longitudinal samples to define evolutionary forces affecting the courses of infection and the driver mutations, whose functions we work to experimentally identify.

## Origins of multicellularity
How different bacterial strains or species coexist and interact when bound together on a surface allows us to explore the origins of multicellular life. We are proud to be part of a [NASA Astrobiology Institute](https://astrobiology.nasa.gov/nai/teams/can-7/umt/) that uses experimental evolution to pursue the goal: 

`.notice` To discover the laws that create Darwin’s ‘tangled bank’ remains one of biology’s grand challenges, one that requires understanding how differences among forms are selected for and how interdependence among forms is enforced.
{: .notice}

## Why genome regions evolve at different rates
In bacterial genomes with multiple chromosomes, smaller, secondary chromosomes evolve more rapidly. We are exploring the root causes of this variation as it may bear on variation in genome stability in all organisms. 

Location matters:

Cooper, V.S., S. Vohr, S. Wrocklage, and P.J. Hatcher. Why Genes Evolve Faster on Secondary Chromosomes in Bacteria. PLoS Computational Biology, 2010.

Flynn, K.M., S.H. Vohr, P.J. Hatcher, and V.S. Cooper. Evolutionary Rates and Gene Dispensability Associate with Replication Timing in the Archaeon Sulfolobus islandicus. Genome Biology and Evolution, 2010.

Morrow, J.D. and Cooper, V.S. Evolutionary Effects of Translocations in Bacterial Genomes. Genome Biology and Evolution, 2012.

How do mutation rates and spectra vary among genome regions?

Intra-genome variation in the rates and spectra of mutations could provide a mechanism by which selection acts on genome organization to influence both the origin of genetic variation as well as its fate.  We use mutation-accumulation experiments (MA) paired with whole-genome-sequencing (WGS) to capture the mutational process of various bacterial species with unprecedented resolution. 


## EvolvingSTEM: evolution-in-action leads to inspired learning.
The real-time evolution of microbes into conspicuous new forms has inspired a high-school curriculum for learning evolutionary biology, ecology, and biotechnology by simple experimentation. Not only do students learn better, they become more engaged in science. We are working to share this curriculum nationwide.
