---
layout: default
title: Research
modified: 2016-02-12
permalink: /research/
---
### Our laboratory seeks to understand how bacterial populations evolve and adapt to colonize hosts and cause disease.
By studying evolution-in-action, in experimental populations and in ongoing infections, using the latest methods in genomic sequencing, we are identifying mechanisms of bacterial adaptation *in vitro* and *in vivo*. 

{% include toc.html %}

## Biofilms
We are focused on how bacteria form complex communities within biofilms and how individual cells perceive cues to attach or disperse.

We have learned that adaptation involves a sequence of mutations in predictable targets that leads to diversification into ecologically differentiated subpopulations. Often, these subpopulations interact synergistically, with certain types affecting host response and others enhancing resistance. In addition, we now routinely sequence the genomes of hundreds of bacterial isolates from longitudinal samples to define evolutionary forces affecting the courses of infection and the driver mutations, whose functions we work to experimentally identify.

## Origins of multicellularity
How different bacterial strains or species coexist and interact when bound together on a surface allows us to explore the origins of multicellular life. We are proud to be part of a [NASA Astrobiology Institute](https://astrobiology.nasa.gov/nai/teams/can-7/umt/) that uses experimental evolution to pursue the goal: 

`.notice` To discover the laws that create Darwin’s ‘tangled bank’ remains one of biology’s grand challenges, one that requires understanding how differences among forms are selected for and how interdependence among forms is enforced.
{: .notice}

## Why genome regions evolve at different rates
In bacterial genomes with multiple chromosomes, smaller, secondary chromosomes evolve more rapidly. We are exploring the root causes of this variation as it may bear on variation in genome stability in all organisms. 

Does chromosome position affect gene evolutionary rate?

Cooper, V.S., S. Vohr, S. Wrocklage, and P.J. Hatcher. Why Genes Evolve Faster on Secondary Chromosomes in Bacteria. PLoS Computational Biology, 2010.

Why many bacteria have multiple chromosomes is largely unknown. We tested the hypothesis that secondary bacterial chromosomes serve as accessory genomes for specific environments by examining its key prediction that genes on secondary chromosomes should evolve faster. By identifying orthologous genes shared by groups of multi-chromosome bacterial genomes, we confirmed that both rates of silent and replacement amino acid substitutions were greater on secondary chromosomes. These increased evolutionary rates result from weaker purifying selection likely because of less frequent gene expression, manifest as reduced use of preferred codons and reduced actual or predicted expression levels. We then asked whether these patterns were inherent to the particular orthologs or effects of chromosome position by studying ortholog families shared by single- and multiple-chromosome genomes. We found that inherently fast-evolving genes tend to be sorted to secondary chromosomes when they arise; however, prolonged evolution on a secondary chromosome greatly accelerated substitution rates. In summary, secondary bacterial chromosomes appear to be intermediates between the conserved, core genome found mostly on primary chromosomes and the transiently necessary accessories found on plasmids. We speculate that the weaker purifying selection experienced by secondary chromosomes could place infrequently expressed genes at risk of mutational decay, a dynamic analogous to the evolution of senescence.

Flynn, K.M., S.H. Vohr, P.J. Hatcher, and V.S. Cooper. Evolutionary Rates and Gene Dispensability Associate with Replication Timing in the Archaeon Sulfolobus islandicus. Genome Biology and Evolution, 2010.

In bacterial chromosomes, the position of a gene relative to the single origin of replication generally reflects its replication timing, how often it is expressed, and consequently its rate of evolution. However, because some archaeal genomes contain multiple origins of replication, bias in gene dosage caused by delayed replication should be minimized and hence the substitution rate of genes should associate less with chromosome position. To test this hypothesis, six archaeal genomes from the genus Sulfolobus containing three origins of replication were selected, conserved orthologs were identified, and the evolutionary rates (dN and dS) of these orthologs were quantified. Ortholog families were grouped by their consensus position and designated by their proximity to one of the three origins (O1, O2, O3). Conserved orthologs were concentrated near the origins and most variation in genome content occurred distant from the origins. Linear regressions of both synonymous and nonsynonymous substitution rates on distance from replication origins were significantly positive, the rates being greatest in the region furthest from any of the origins and slowest among genes near the origins. Genes near O1 also evolved faster than those near O2 and O3, which suggests that this origin may fire later in the cell cycle. Increased evolutionary rates and gene dispensability are strongly associated with reduced gene expression caused in part by reduced gene dosage during the cell cycle. Therefore, in Archaea as well as in Bacteria, evolutionary rates and variation in genome content associate with replication timing.

Morrow, J.D. and Cooper, V.S. Evolutionary Effects of Translocations in Bacterial Genomes. Genome Biology and Evolution, 2012.

It has become clear that different genome regions need not evolve uniformly. This variation is particularly evident in bacterial genomes with multiple chromosomes, in which smaller, secondary chromosomes evolve more rapidly. We previously demonstrated that substitution rates and gene dispensability were greater on secondary chromosomes in many bacterial genomes. In Vibrio, the secondary chromosome is replicated later during the cell cycle, which reduces the effective dosage of these genes and hence their expression. More rapid evolution of secondary chromosomes may therefore reflect weaker purifying selection on less expressed genes. Here, we test this hypothesis by relating substitution rates of orthologs shared by multiple Burkholderia genomes, each with three chromosomes, to a study of gene expression in genomes differing by a major reciprocal translocation. This model predicts that expression should be greatest on chromosome 1 (the largest) and least on chromosome 3 (the smallest) and that expression should tend to decline within chromosomes from replication origin to terminus. Moreover, gene movement to the primary chromosome should associate with increased expression, and movement to secondary chromosomes should result in reduced expression. Our analysis supports each of these predictions, as translocated genes tended to shift expression toward their new chromosome neighbors despite inevitable cis-acting regulation of expression. This study sheds light on the early dynamics of genomes following rearrangement and illustrates how secondary chromosomes in bacteria may become evolutionary test beds.

How do mutation rates and spectra vary among genome regions?

Intra-genome variation in the rates and spectra of mutations could provide a mechanism by which selection acts on genome organization to influence both the origin of genetic variation as well as its fate.  We use mutation-accumulation experiments (MA) paired with whole-genome-sequencing (WGS) to capture the mutational process of various bacterial species with unprecedented resolution. 



## EvolvingSTEM: evolution-in-action leads to inspired learning.
The real-time evolution of microbes into conspicuous new forms has inspired a high-school curriculum for learning evolutionary biology, ecology, and biotechnology by simple experimentation. Not only do students learn better, they become more engaged in science. We are working to share this curriculum nationwide.
